src/test/java/uk/gov/pay/ledger/rule/AppWithPostgresAndSqsRule.java

```diff
@@ -18,22 +18,30 @@
 public class AppWithPostgresAndSqsRule extends ExternalResource {
     private static String CONFIG_PATH = resourceFilePath("config/test-config.yaml");
     private final Jdbi jdbi;
-    private final AmazonSQS sqsClient;
+    private AmazonSQS sqsClient;
     private PostgreSQLContainer postgres;
     private DropwizardAppRule<LedgerConfig> appRule;
 
     public AppWithPostgresAndSqsRule() {
+        this(false);
+    }
+
+    public AppWithPostgresAndSqsRule(boolean enableBackgroundProcessing) {
```

Comment:
We could use config override which give us flexibility to override any variable. See example
https://github.com/alphagov/pay-connector/blob/84860a0eb8bd364fd280b4a6440f4792e9bfc6ff/src/test/java/uk/gov/pay/connector/rules/AppWithPostgresRule.java#L37

Reply:
good suggestion

Reply:
Updated